hazelcast-oss/Dockerfile

```diff
@@ -45,7 +45,9 @@ RUN echo "Upgrading APK packages" \
     && chmod -R +r ${HZ_HOME} \
     && echo "Cleaning APK packages and redundant files/folders" \
     && apk del libxml2-utils zip unzip \
-    && rm -rf /var/cache/apk/* ${HZ_HOME}/get-hz-dist-zip.sh ${HZ_HOME}/hazelcast-distribution.zip ${HZ_HOME}/tmp
+    && rm -rf /var/cache/apk/* ${HZ_HOME}/get-hz-dist-zip.sh ${HZ_HOME}/hazelcast-distribution.zip ${HZ_HOME}/tmp \
+    # unzip may not preserve permissions. Give execute permission to scripts
```

Comment:
this is mostly issue in windows so perhaps change comment e.g.

# Give execute permission to scripts to overcome issue with permissions not being properly propagated on Windows OS

Reply:
rev : d74b3bc717b3b80a46782357f1be9a0c32c04621

Reply:
The comment is 
`Grant execute permission to scripts in order to address the issue of permissions not being accurately propagated on Windows OS`

Reply:
perfect. thanks